js/playerMovement.js

```diff
@@ -0,0 +1,23 @@
+  function charMoveLeft(){
```

Comment:
Your movement functions are called inside of callback function of the keyborad event listener.
Then what if I put var position = $('#player').position() line to line 4 of gameplay.js?